src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml

```diff
@@ -60,7 +60,7 @@ config:
         user_info: 'Mon compte'
         password: 'Senhal'
         rules: "Règlas d'etiquetas automaticas"
-        # ignore_origin: 'Ignore origin rules'
+        ignore_origin: 'Règlas d’ignorar l’origina'
```

Comment:
You need to add a backslash before the `'`
Or do like that: 

```
ignore_origin: "Règlas d’ignorar l’origina"
```

Reply:
Nope, `'` and `’` the same quote, so it's fine.

Reply:
Absolutely, I am more into the curly one :)